site2/website/versioned_sidebars/version-2.7.1-sidebars.json

```diff
@@ -150,7 +150,6 @@
     "Reference": [
       "version-2.7.1-reference-terminology",
       "version-2.7.1-reference-cli-tools",
-      "version-2.7.1-pulsar-admin",
```

Comment:
This will remove the pulsar-admin cli page from the website, right? I think it would be really valuable to leave this page in place and to put a banner at the top to indicate the page is deprecated and that there is a new/updated pulsar admin cli website. As a user, I'd be surprised if this page would just suddenly disappear because I didn't know it was out of date or that there was a new/different place to look for the pulsar admin cli documentation.

Reply:
We already have a "button" at the top navigation since the beginning. It is easy to find and use.
![image](https://user-images.githubusercontent.com/50226895/112555716-ae955380-8e03-11eb-9fd0-bd524668694b.png)

Reply:
Thanks for pointing out that documentation. I agree that it will be easy for new users to find and use.

Although, users who were using the old pulsar admin client cli documentation might be confused when that portion of the website suddenly disappears. I would have been one of those users. I had been using that section of the documentation without realizing that the top level cli client documentation existed.

While I had seen the pulsar admin cli tool website, I didn't know that it was more up to date, so I had continued to use the older version. A banner or some type of redirection to the new website could help users make the transition to the right documentation. I think this is ultimately a great change, but I also want to ease transition for users that were used to the old documentation.